webroot/js/app/modules/index.js

```diff
@@ -0,0 +1,164 @@
+
+
+/**
+ * Templates that uses this component (directly or indirectly)
+ *
+ * Template/Modules/index.twig
+ * - Element/Toolbar/filter.twig
+ * - Element/Toolbar/pagination.twig
+ *
+ */
+
+Vue.component('modules-index', {
+
+    /**
+     * component properties
+     *
+     * @returns {Object}
+     */
+    data() {
+        return {
+            searchQuery: '',
+            pageSize: '100',
+            page: '',
+            sort: '',
+        };
+    },
+
+    created() {
+        // load url params when component initialized
+        this.loadUrlParams();
+    },
+
+    /**
+     * component methods
+     */
+    methods: {
+
+        /**
+         * extract params from page url
+         *
+         * @returns {void}
+         */
+        loadUrlParams() {
```

Comment:
`loadUrlParams` e  `buildUrlParams` rendono deprecati/inutili i metodi `LinkHelper->page`, `LinkHelper->pageSize` e `LinkHelper->query`? In tal caso valutiamo se eliminarli. Io avrei preferito demandare all'helper la gestione dei link, ma non so se usando `vue` ciò è possibile.

Reply:
yes, è vero, non avevo visto, meglio discuterne